Example/Backpack/Stories/ChipStory.swift

```diff
@@ -0,0 +1,49 @@
+//
```

Comment:
extraneous line

Reply:
I'm so used to outsourcing my brain to eslint 😂